MapboxCoreNavigation/MBRouteController.h

```diff
@@ -17,3 +17,5 @@ extern NSString *const MBRouteControllerWillReroute;
 extern NSString *const MBRouteControllerDidReroute;
 extern NSString *const MBRouteControllerDidFailToReroute;
 extern NSString *const MBRouteControllerDidFindFasterRouteKey;
+extern NSString *const MBErrorDomain;
+extern NSString *const MBSpokenInstructionErrorCode;
```

Comment:
These constants (like all the others here) need documentation comments so that they appear in the jazzy documentation.